docs/getting_started/basics/contexts.md

```diff
@@ -0,0 +1,15 @@
+# Context lookup
+
+As can be found in other sections, the configuration needs contexts to work. _Context_ itself is a concept in JSON-LD.
+
+Despite that contexts are URLs, components.js does not always perform HTTP requests to fetch them, for several concerns. Instead, components.js will first attempt to perform _local context lookup_, and will fall back to retrieve the context from the URL if local context lookup fails.
+
+For local context lookup, components.js will check inside all modules under `node_modules` directory to see if they contain the relevant context file; it will also try to find the context file from the _current_ module/project.
```

Comment:
```suggestion
For local context lookup, Components.js will check inside all modules under the `node_modules/` directory to see if they contain the relevant context file; it will also try to find the context file from the _current_ module/project.
```